File: src/main/Java/boj_step3/boj2739.java
import java.io.*;

public class boj2739 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        int n = Integer.parseInt(reader.readLine());
        for (int i = 1; i < 10; i++) writer.append(n + " * " + i + " = " + n * i + "\n");
        writer.close();
    }
}

File: src/main/kotlin/boj 2739.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val n = r.readLine()!!.toInt()
    for (i in 1..9) w.write(n.toString() + " * " + i.toString() + " = " + (n * i).toString() + "\n")
    w.close()
}